File: backend/src/main/java/gestion_inventarios/backend/domain/model/locations/LocationType.java
package gestion_inventarios.backend.domain.model.locations;

public enum LocationType {

    WAREHOUSE("Bodega"),
    STORE("Tienda"),
    VIRTUAL("Virtual");

    private final String displayName;

    LocationType(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: frontend/src/app/core/models/location/LocationType.ts
export enum LocationType {
  WAREHOUSE = 'WAREHOUSE',
  STORE = 'STORE',
  VIRTUAL = 'VIRTUAL'
}

export const LOCATION_TYPE_LABELS: Record<LocationType, string> = {
  [LocationType.WAREHOUSE]: 'Bodega',
  [LocationType.STORE]: 'Tienda',
  [LocationType.VIRTUAL]: 'Virtual'
};
